target/cheri-common/cheri-lazy-capregs.h

```diff
@@ -334,6 +336,14 @@ static inline target_ulong get_capreg_tag_revokable(CPUArchState *env, unsigned
     return tagged;
 }
 
+#else
+
+static inline target_ulong get_capreg_tag_revokable(CPUArchState *env, unsigned regnum) {
```

Comment:
How about moving the #ifdef TARGET_MIPS into the implementation?

Reply:
Done.